SIZES.md

```diff
@@ -76,6 +77,8 @@
 | indexed | 40 B | n/a B | n/a B |
 | indexOf | 110 B | 1570 B | -1460 B |
 | init | 152 B | 493 B | -341 B |
+| insert | 124 B | 378 B | -254 B |
```

Comment:
what's that? an below as well

Reply:
PR's of @Kelin2025 has no size calculations

Reply:
Which one? Please leave a link to it

Reply:
#26, #27, #30, #31

Reply:
thanks